chart/values.yaml

```diff
@@ -96,18 +96,20 @@ iq_server:
   applicationServiceAnnotations:
   # Annotations for the admin service
   adminServiceAnnotations:
+  # Annotations to apply to the DB migration job
+  migrationJobAnnotations:
```

Comment:
Could you please add a small test for this in `iq-server-jobs_test.yaml` to check setting this value applies the annotations correctly